src/main/resources/generate_from_proto.sh

```diff
@@ -4,8 +4,23 @@
 
 set -e
 
-python3 -m grpc_tools.protoc -I../proto \
---python_out=../python/pb --grpc_python_out=../python/pb \
-../proto/service.proto \
-../proto/github.com/gogo/protobuf/gogoproto/gogo.proto \
-../proto/gopkg.in/bblfsh/sdk.v1/uast/generated.proto
+# Python server
+
+pip3 install grpcio==1.10.0 grpcio-tools==1.10.0
+python3 -m grpc_tools.protoc -Isrc/main/proto \
```

Comment:
As soon as it's an interactive script meant to be run by developer on .proto updates, do you think we could have few very simple logs with `echo` in here?

I find scripts that provide small bits of the output very useful for debugging, i.e

```
Installing grpcio
Generating Python code from .proto to src/main/python/pb
scalabp binary is not found, installing from YYY
Generating Scala code from .proto to src/main/scala
```

Reply:
done